src/app/shared/components/hmcts-em-viewer-ui/viewers/annotation-pdf-viewer/rotation-toolbar/rotation.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input } from '@angular/core';
+import { PdfRenderService } from '../../../data/pdf-render.service';
+import { EmLoggerService } from '../../../logging/em-logger.service';
+
+
+@Component({
+    selector: 'app-rotation-toolbar',
+    templateUrl: './rotation.component.html',
+    styleUrls: ['./rotation.component.scss'],
+    providers: []
+})
+export class RotationComponent {
+    @Input() pageNumber: number;
+
+    constructor(private pdfRenderService: PdfRenderService,
+                private log: EmLoggerService) {
+        this.log.setClass('RotationComponent');
+    }
+
+    calculateRotation(rotateVal): number {
+        if (rotateVal >= 0) {
+            return (rotateVal >= 360) ? rotateVal - 360 : rotateVal;
+        } else {
+            return 360 - Math.abs(rotateVal);
+        }
+    }
```

Comment:
I believe this whole method can become simple with the use of mod. whether its a minus or greater than 360